README.md

```diff
@@ -322,102 +289,132 @@ The `importpath` import path must always be specified. If urls are specified, it
     <tr>
       <td><code>importpath</code></td>
       <td>
-        <code>String, optional</code>
-        <p>An import path in Go, which also provides a default value for the
-        root of the target remote repository</p>
-      </td>
-    </tr>
-    <tr>
-      <td><code>remote</code></td>
-      <td>
-        <code>String, optional</code>
-        <p>The URI of the target remote repository, if this cannot be determined
-        from the value of <code>importpath</code>.</p>
+        <code>String, required</code>
+        <p>The root import path for libraries in the repository.</p>
       </td>
     </tr>
     <tr>
-      <td><code>vcs</code></td>
+      <td><code>commit</code></td>
       <td>
         <code>String, optional</code>
-        <p>The version control system to use for fetching the repository. Useful
-        for disabling importpath redirection if necessary.</p>
+        <p>The commit hash to checkout in the repository.<br>
+        Either <code>commit</code> or <code>tag</code> must be specified.</p>
```

Comment:
maybe `Exactly one of <code>commit</code> or <code>tag</code> must be specified.`
also the same below